site/src/pages/ManagementSettingsPage/SidebarView.tsx

```diff
@@ -148,11 +148,12 @@ const DeploymentSettingsNavigation: FC<DeploymentSettingsNavigationProps> = ({
 							Users
 						</SidebarNavSubItem>
 					)}
-					{experiments.includes("notifications") && (
+					<Stack direction={"row"} alignItems={"center"} css={{ gap: 0 }}>
```

Comment:
Nit: When passing string values directly you can pass the value directly without the brackets:
```
direction="row"
```